service/src/main/java/io/camunda/service/entities/FlownodeInstanceEntity.java

```diff
@@ -7,4 +7,20 @@
  */
 package io.camunda.service.entities;
 
-public record FlownodeInstanceEntity() {}
```

Comment:
I agree with @houssain-barouni ove renaming this class to `FlowNodeInstanceEntity`.